Python/pytime.c

```diff
@@ -1059,7 +1059,7 @@ _PyTime_localtime(time_t t, struct tm *tm)
     return 0;
 #else /* !MS_WINDOWS */
 
-#ifdef _AIX
+#ifdef defined(_AIX) and (SIZEOF_LONG ==4)
```

Comment:
I don't think that `#ifdef` is used properly here. Do you mean the following code?

```suggestion
#if defined(_AIX) and (SIZEOF_LONG == 4)
```

Is it possible to use SIZEOF_TIME_T rather than SIZEOF_LONG?

Reply:
Will check on SIZEOF, and make #if change in any case.